apps/aecore/src/aec_coinbase_tx.erl

```diff
@@ -21,15 +21,17 @@
 
 -include("common.hrl").
 
--record(coinbase_tx, {account   = <<>> :: pubkey()}).
+-record(coinbase_tx, {account       = <<>> :: pubkey(),
+                      block_height  = -1 :: integer()}).
```

Comment:
May I suggest stronger type (proper height - not negative - so Dialyzer checks stronger)? Default may be omitted too (that is my preference) though I understand it is debatable.